



Loading checkpoint shards: 100%|██████████████████████████████████████████████████████████████████████████████████| 4/4 [00:21<00:00,  5.28s/it]
2024-08-28 16:31:37,740 - WARNING - A <class 'transformers.models.llama.modeling_llama.LlamaForCausalLM'> model is loaded from 'meta-llama/Meta-Llama-3.1-8B-Instruct', and no v_head weight is found. This IS expected if you are not resuming PPO training.
Loading checkpoint shards: 100%|██████████████████████████████████████████████████████████████████████████████████| 4/4 [00:02<00:00,  1.42it/s]
2024-08-28 16:31:41,603 - WARNING - A <class 'transformers.models.llama.modeling_llama.LlamaForCausalLM'> model is loaded from 'meta-llama/Meta-Llama-3.1-8B-Instruct', and no v_head weight is found. This IS expected if you are not resuming PPO training.
2024-08-28 16:31:41,969 - INFO - Sender dataset size: 1000
2024-08-28 16:31:41,969 - INFO - Receiver dataset size: 1000
2024-08-28 16:31:41,987 - WARNING - Detected kernel version 4.18.0, which is below the recommended minimum of 5.5.0; this can cause the process to hang. It is recommended to upgrade the kernel to the minimum version or higher.
2024-08-28 16:31:44,014 - WARNING - Detected kernel version 4.18.0, which is below the recommended minimum of 5.5.0; this can cause the process to hang. It is recommended to upgrade the kernel to the minimum version or higher.
Batch:   0%|                                                                                                             | 0/31 [00:00<?, ?it/s]
Epoch:   0%|                                                                                                             | 0/10 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "/g/g10/smith585/codebases/collaborative-stegosystem/experiments/18_simple_ppo.py", line 90, in <module>
    query_tensors = batch["input_ids"].to(sender_model.pretrained_model.device)
                    ^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'list' object has no attribute 'to'